src/Resources/config/services/providers.xml

```diff
@@ -53,7 +53,6 @@
             id="Sylius\RefundPlugin\Provider\RelatedPaymentIdProviderInterface"
             class="Sylius\RefundPlugin\Provider\DefaultRelatedPaymentIdProvider"
         >
-            <argument type="service" id="sylius.repository.order" />
         </service>
```

Comment:
So it could be shorten:
```xml
<service
   id="Sylius\RefundPlugin\Provider\RelatedPaymentIdProviderInterface"
   class="Sylius\RefundPlugin\Provider\DefaultRelatedPaymentIdProvider"
/>
```